picard/formats/asf.py

```diff
@@ -228,7 +228,9 @@ def _remove_deleted_tags(self, metadata, tags):
                     del tags[real_name]
 
     def supports_tag(self, name):
-        return name in self.__TRANS
+        return (name in self.__TRANS
+                or name in ('~rating', '~length', 'totaldiscs', 'totaltracks')
```

Comment:
Do you know to which extent totaltracks is supported by ASF? It's not listed as supported on https://picard.musicbrainz.org/docs/mappings/ and I can't see anything that loads or saves it.